shared/message/behaviour/SoccerObjectPriority.proto

```diff
@@ -29,7 +34,7 @@ message SoccerObjectPriority {
         GOAL_RIGHT              = 4;
         GROUND_LEFT             = 5;
         GROUND_RIGHT            = 6;
-        OTHE                    = 7;
+        OTHER                   = 7;
     }
 
     int32      ball       = 1;
```

Comment:
What do these even mean

Reply:
Not sure. `GOAL_SEARCH` and `LOST` are the ones used in the codebase, but they seem fairly self-explanatory

Reply:
Sorry, maybe should have put this comment a bit lower. I meant the ball, goal and line ints.